src/app/sign-in/components/form-sign-in/form-sign-in.component.ts

```diff
@@ -191,14 +196,24 @@ export class FormSignInComponent implements OnInit, AfterViewInit {
     this._oauthService
       .updateOauthSession(value)
       .subscribe((requestInfoForm) => {
-        this._router.navigate(['/oauth/authorize'], {
-          queryParams: {
-            client_id: this.signInLocal.params.client_id,
-            response_type: this.signInLocal.params.response_type,
-            scope: this.signInLocal.params.scope,
-            redirect_uri: this.signInLocal.params.redirect_uri,
-          },
-        })
+        this._gtag
+          .reportEvent('RegGrowth', 'Sign-In', requestInfoForm)
+          .subscribe(
+            () => {
+              this._router.navigate(['/oauth/authorize'], {
+                queryParams: {
+                  client_id: this.signInLocal.params.client_id,
+                  response_type: this.signInLocal.params.response_type,
+                  scope: this.signInLocal.params.scope,
+                  redirect_uri: this.signInLocal.params.redirect_uri,
+                },
+              })
+            },
+            () => {
+              // TODO @DanielPalafox display error using a toaster
```

Comment:
let's handle the error in the same way as the success for analytics events, as you do here 

https://github.com/ORCID/orcid-angular/blob/effa00ba6f3faa3c1d785bb4815fb6749b566e7d/src/app/sign-in/components/form-sign-in/form-sign-in.component.ts#L118-L120

this  since we want to keep things moving even if analytics is down

Reply:
I think we don't even want to show the toaster when analytics fails

Reply:
Make sense. I will change it